dashboard/api_test.py

```diff
@@ -1913,6 +1917,18 @@ def test_calculate_expired(users_without_with_cache, enrollment, certificate, cu
     cache.save()
 
     expected = needs_update + [up_to_date[0]] if expired else needs_update
+    assert sorted(api.calculate_users_to_refresh_in_bulk()) == sorted([user.id for user in expected])
+
+
+@patch("dashboard.api.CACHE_KEY_FAILED_USERS_NOT_TO_UPDATE", SET_USER_IDS_NOT_TO_UPDATE)
```

Comment:
Can we do this with a fixture instead? The fixture should patch the two constants and also clear the values beforehand so that it's always empty when the test runs